src/md/contributing.md

```diff
@@ -124,17 +145,15 @@ Thanks for contributing!
 
 ## Merging a Pull Request (committers)
 
-1. Ensure that the basic requirements for a pull request are met. This includes:
+1. Ensure that basic requirements for a pull request are met. This includes:
+  - PR was suffiently reviewed and comments were addressed
+  - there was sufficient time for others to review
```

Comment:
Please include some suggestions discussed in the thread (and mentioned in the jira) for this point such as suggested protocol for waiting for other reviewers..
https://lists.apache.org/thread.html/40448aebada13fbd15ec88525b7edba13fb95095eae18c0018ddcda4@%3Cdev.apex.apache.org%3E

Reply:
Do you mean this:

> main reviewer has to wait till all the other reviewers say
something like LGTM or a timeout period like 2 days before merging it

If anything else, please suggest the text change.

Reply:
IMO, Apache voting policy should apply here and we don't need to invent something new. Any -1 must be explicitly withdrawn, everything else can be handled by a grace/timeout period.

Reply:
@tweise yes, it is to alleviate the concern of premature merging before others who are reviewing have a chance to finish